src/core/reducers/addons.js

```diff
@@ -30,6 +30,12 @@ export default function addon(state = initialState, action) {
         newState[key] = {
           ...thisAddon,
           ...thisAddon.theme_data,
+          // TODO: Remove this when
+          // https://github.com/mozilla/addons-frontend/issues/1416 is fixed.
+          // theme_data will contain `description: 'None'` when the description
```

Comment:
I've filed https://github.com/mozilla/addons-server/issues/5650 but I suspect it might be a requirement because we put that object in the weird `data-browsertheme` or whatever. Still, filed! I'll add it to the comment.